Sources/Client/Model/Channel/Channel.swift

```diff
@@ -116,6 +118,8 @@ public final class Channel: Codable {
     /// An event when the channel was updated.
     public var onUpdate: OnUpdate<Channel>?
     
+    let needsToCreate: Bool
```

Comment:
I suggest renaming to `createOnBackend` or `needsCreation` or just `createIfNecessary`

Reply:
it used like that:
```
if channel.needsToCreate {
    // send channel data
}
```
I don't like `Backend` word.
`needsCreation` is the same as `needsToCreate`
`createIfNecessary` has other meaning

Reply:
`needsToCreate` is grammatically wrong though, `Channel` does not create anything, it is created [by backend]. `needsCreation` is the correct one, and `createOnBackend` has more context.